study-builder/studies/rarex/study.conf

```diff
@@ -576,34 +606,79 @@
                 },
                 {
                   "type": "TEXT",
-                  "field": "last_name",
+                  "field": "unable_first_name_1",
                   "activityCode": "CONSENT_ASSENT",
-                  "questionStableId": "PARENTAL_ASSENT_PARENT_LASTNAME"
+                  "questionStableId": "PARENTAL_ASSENT_CHILD_FIRSTNAME"
                 },
                 {
                   "type": "TEXT",
-                  "field": "first_name",
+                  "field": "unable_last_name_1",
                   "activityCode": "CONSENT_ASSENT",
-                  "questionStableId": "PARENTAL_ASSENT_PARENT_FIRSTNAME"
+                  "questionStableId": "PARENTAL_ASSENT_CHILD_LASTNAME"
                 },
                 {
                   "type": "TEXT",
-                  "field": "child_last_name",
+                  "field": "unable_signature_1",
                   "activityCode": "CONSENT_ASSENT",
-                  "questionStableId": "PARENTAL_ASSENT_CHILD_LASTNAME"
+                  "questionStableId": "CONSENT_ASSENT_PARENT_SIGNATURE"
+                },
+                {
+                  "type": "ACTIVITY_DATE",
+                  "field": "date",
+                  "activityCode": "CONSENT_ASSENT"
                 },
                 {
                   "type": "TEXT",
-                  "field": "child_first_name",
+                  "field": "person_first_name_1",
```

Comment:
Should these two be called `parent_first_name` and `parent_last_name`?

Reply:
Actually no, they are configured as pdf variables themselves to be named as `person_first_name_1` & `person_last_name_1` for the first encounter and `person_first_name_2` & `person_last_name_2` for the second encounter